### PR TITLE
VDX-298: Do not show deeplink button on desktop browser.

### PR DESCRIPTION
**Pull request by sanderPostma**
Alse removed logo from narrow screens for now because it looked corrupted. (We need a mobile compatibel logo + container first.)

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/components/AuthenticationQR/index.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ class AuthenticationQR extends Component<AuthenticationQRProps> {
   render() {
     // Show the loader until we have details on which parameters to load into the QR code
     return this.state.qrCode ? (
-      <NonMobileOS><div>{this.state.qrCode}</div></NonMobileOS>
+      <div>{this.state.qrCode}</div>
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/components/AuthenticationQR/index.tsx:114]: not sure about this approach. demo is cross-device

File: packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx [hunk 2]
```diff
@@ -153,12 +153,12 @@ const SSICredentialIssueRequestPage: React.FC = () => {
                         marginTop: isNarrowScreen ? 20 : '15%',
                         alignItems: 'center'
                     }}>
-                        <NonMobile>
+                        <NonMobileOS>
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx:156]: I preferred the previous approach here to use a cutoff point for the width of the screen. we can decrease that. but that's my preference.

File: packages/oid4vci-demo-frontend/src/components/AuthenticationQR/index.tsx (hunk 1 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/components/AuthenticationQR/index.tsx:114]: I moved control of whether to display the QR code or the deep link to the parent page, the QR code doesn't have to be rendered at all on mobile devices.

File: packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx (hunk 2 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx:156]: Showing a deeplink on a desktop is pointless. The deeplink is not going to work on a desktop device and a QR code is not going to work on a mobile device unless you have two phones.
 So the OS type controls which one of the two is displayed.

**Pull request merged**